trailpanion/src/components/App.js

```diff
@@ -6,20 +6,24 @@ import PlaceDetails from "./PlaceDetails";
 import TrailResults from "./TrailResults";
 import ResultsMap from "./ResultsMap";
 import GearList from "./GearList";
+import { store } from "../data/store"
+import { Provider } from "react-redux"
 
 const App = () => {
   return (
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<SharedLayout />}>
-          <Route index element={<SearchAndMapControl />} />
-          <Route path="resultsMap" element={<ResultsMap />} />
-          <Route path="placeDetails" element={<PlaceDetails />} />
-          <Route path="trailResults" element={<TrailResults />} />
-          <Route path="gearList" element={<GearList />} />
-        </Route>
-      </Routes>
-    </BrowserRouter>
+    <Provider store={store}>
```

Comment:
Your app must be wrapped with a redux provider so that your components can access the store